android/src/main/java/com/mapbox/mapboxgl/MapboxMapController.java

```diff
@@ -1095,7 +1099,8 @@ public void onActivityDestroyed(Activity activity) {
 
   @Override
   public void setCameraTargetBounds(LatLngBounds bounds) {
-    mapboxMap.setLatLngBoundsForCameraTarget(bounds);
+    this.bounds = bounds;
+//    mapboxMap.setLatLngBoundsForCameraTarget(bounds);
```

Comment:
We do want to still allow changing the bounds at runtime (when the map is already active). Wouldn't be the following logic more appropriate? 

```
this.bounds = bounds; 
if(mapboxMap != null) {
   mapboxMap.setLatLngBoundsForCameraTarget(bounds);
}
```